Sources/TuistGenerator/Generator/BuildPhaseGenerator.swift

```diff
@@ -715,6 +724,42 @@ final class BuildPhaseGenerator: BuildPhaseGenerating {
         embedXPCServicesBuildPhase.files = pbxBuildFiles
     }
 
+    func generateEmbedPlugInsBuildPhase(
+        path: AbsolutePath,
+        target: Target,
+        graphTraverser: GraphTraversing,
+        pbxTarget: PBXTarget,
+        fileElements: ProjectFileElements,
+        pbxproj: PBXProj
+    ) throws {
+        let targetDependencies = graphTraverser.directLocalTargetDependencies(path: path, name: target.name).sorted()
+        let plugIns = targetDependencies.filter { $0.target.isEmbeddablePlugin() }
```

Comment:
nit: it should _plugin_ and _plugIn_. I find the casing a bit weird. Would you mind changing that?

Reply:
Good catch! I hadn’t noticed that. I’ll switch to *plugin* for consistency.